core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -86,6 +98,21 @@ public void warn(String msg, Object... args) {
     }
 
     /**
+     * Will format the given message with the given arguments and prints it on standard output with the prefix:
+     * "DEBUG: Smart-Testing - ", if debug mode is enabled.
+     *
+     * @param msg
+     *     The string message (or a key in the message catalog)
+     * @param args
+     *     arguments to the message
+     */
+    public void debug(String msg, Object... args) {
+        if (isDebugEnabled()) {
+            System.out.println(getFormattedMsg("DEBUG", msg, args));
```

Comment:
Shouldn't be this a call to `logger` instead of `sysout`?

Reply:
This is our logger and I remember @MatousJobanek did that by purpose to go to sysout too.